unverified/sexpr-bootstrap/x64/32/Holmakefile

```diff
@@ -16,7 +14,7 @@ README.md: $(CAKEMLDIR)/developers/readme_gen readmePrefix $(patsubst %,%readmeP
 cake-sexpr-$(ARCH)-$(WORD_SIZE): *$(ARCH)SexprScript.sml
 
 cake.S: cake-sexpr-$(ARCH)-$(WORD_SIZE) $(CAKE_BIN)
-	$(CAKE_BIN) --sexp=true --exclude_prelude=true --skip_type_inference=true --emit_empty_ffi=true --heap_size=$(HEAP_MB) --stack_size=$(STACK_MB) --reg_alg=0 <$< >$@
+	CML_STACK_SIZE=8000 CML_HEAP_SIZE=8000 $(CAKE_BIN) --sexp=true --exclude_prelude=true --skip_type_inference=true --emit_empty_ffi=true --reg_alg=0 <$< >$@
```

Comment:
could these numbers (8000) not still be Holmakefile variables (as previously)? also why did they change?

Reply:
This 8000 here is specifically to set the CakeML compiler's heap and stack (of $(CAKE_BIN)) when bootstrapping itself using the sexpression representation.

Reply:
I guess @xrchz means you could still write
```
CML_STACK_SIZE = 8000

cake.S: ....
    CML_STACK_SIZE=$(CML_STACK_SIZE) ....
```

Perhaps it'd be nicer still if Holmake supported the `export` directive that normal `make` has; then you'd write

```
export CML_STACK_SIZE = 8000
```

and wouldn't need to use the prefix in the recipe at all.